src/gpodder/gtkui/shownotes.py

```diff
@@ -21,8 +21,18 @@
 from gi.repository import Gdk
 from gi.repository import GObject
 from gi.repository import Pango
+
 import os
+import urllib.parse
 
+has_webkit2 = False
+try:
+    import gi
+    gi.require_version('WebKit2', '4.0')
```

Comment:
If it's not installed, this throws a `ValueError`, so this might need to be caught in addition to the `ImportError` (this is on Python 2, but I expect it to behave the same on Python 3):

```
% python2
Python 2.7.13 (default, Jan 16 2017, 14:43:29) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import gi
>>> gi.require_version('WebKit2', '4.0')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/gi/__init__.py", line 118, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace WebKit2 not available
```